src/main/java/org/example/ApiServiceConnections/ApiConnectionConstans.java

```diff
@@ -0,0 +1,12 @@
+package org.example.ApiServiceConnections;
+
+public class ApiConnectionConstans {
+    protected static final String URL = "https://www.dnd5eapi.co/api/classes/";
+
+    public static final String[] classes = {"barbarian", "bard", "cleric", "druid", "fighter",
+            "monk", "paladin", "ranger", "rogue", "sorcerer", "warlock", "wizard"};
+    protected static final String[] races = {"dragonborn", "dwarf", "elf", "gnome", "half-elf",
```

Comment:
Please put the variable name in upper case

Reply:
Done!